docs/topics/secured-mission-deploy-rhsso.adoc

```diff
@@ -1,42 +1,42 @@
 [[mission-secured-deploy-rhsso]]
 = Deploy RH SSO to your {OpenShiftLocal}
 
-Before you can use your {OpenShiftLocal}, you need to have it installed, configured, and running. You can find details on installing a {OpenShiftLocal} for your platform in link:{link-openshift-local-guide}[{openshift-local-guide-name}].
-
+. Before you can use your {OpenShiftLocal}, you need to have it installed, configured, and running. You can find details on installing a {OpenShiftLocal} for your platform in link:{link-openshift-local-guide}[{openshift-local-guide-name}].
++
 NOTE: You only need to do this once for any of the secured booster missions. They can share the same RH SSO/{OpenShiftLocal} setup.
 
-NOTE: If you have already run the minishift command with a different memory setting/iso-url value, you need to stop it and completely delete the ~/.minishift directory
-before running the following startup sequence.
-
-NOTE: You must use a different boot iso image than the default in order for RH SSO to startup:
 
+. Delete the current {OpenShiftLocal} configuration and restart it with a new ISO.
++
+IMPORTANT: You must use a different boot iso image than the default in order for RH SSO to startup. If you have already run the `minishift` command with a different memory setting and iso-url value, you need to stop it and completely delete the `~/.minishift` directory before running the following startup sequence.
++
 [source,bash,options="nowrap",subs="attributes+"]
 ----
-minishift start --memory=6000 --vm-driver=virtualbox --iso-url=https://github.com/minishift/minishift-centos-iso/releases/download/v1.0.0-rc.4/minishift-centos7.iso
+$ minishift delete
+$ rm -r ~/.minishift
+$ minishift start --memory=6000 --vm-driver=virtualbox --iso-url=https://github.com/minishift/minishift-centos-iso/releases/download/v1.0.0-rc.4/minishift-centos7.iso
```

Comment:
There's a `v1.0.0` "final" version already, we should point to that instead of rc4: https://github.com/minishift/minishift-centos-iso/releases